### DISCUSSION TITLE
Converting item-list prediction to original item-id in the ETL work.

### DISCUSSION THREAD

**Discussion by swapnilpanda** [NVIDIA-Merlin/Transformers4Rec#704] [Q&A]
I have been going through the example end-to-end prediction on transformer4rec, in the ETL notebook the item-id was shown in 6 digits, 
![image](https://github.com/NVIDIA-Merlin/Transformers4Rec/assets/56393206/94bbfed7-53c0-4f57-9a66-8ffa7d46b46c)
however in the inference part, the item-id list we got had the item-id in 1 to 4 digit.
![image](https://github.com/NVIDIA-Merlin/Transformers4Rec/assets/56393206/68f4581c-a891-41cf-bff7-8d48aac4a9d1)
 How do I track back to the original item-id so that I can actually make the inference and check my results? 
In other words, how is the item id before and after transformation are mapped?
Any help will be appreciated, thanks.
